### DISCUSSION TITLE
This's a request for add language options

### DISCUSSION THREAD

**Discussion by kasen-zhang** [immich-app/immich#6471] [Feature Request]
### The feature

Can language options be provided on the server side

### Platform

- [x] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
What do you mean by "language options"?

**Reply by kasen-zhang** [reply]
I'm sorry, I'm not a native English speaker, and my English is not very good. What I mean is “A self-selectable option for region/language allows users to choose their preferred language or regional settings based on their needs and preferences. This design intention is to provide a better user experience and cater to the varying needs and preferences of different users”

**Comment by bo0tzz** [comment]
Duplicate of #1688
